app/views/state_file/questions/az_charitable_contributions/edit.html.erb

```diff
@@ -26,8 +26,9 @@
       </div>
     </div>
 
-    <button class="button button--primary" type="submit">
-      <%= t("general.continue") %>
-    </button>
+    <% unless params[:return_to_review].nil? %>
+      <%= hidden_field_tag "return_to_review", params[:return_to_review] %>
+    <% end %>
+    <%= f.continue %>
```

Comment:
these are a little more verbose now because we don't want to include the tag at all if the return_to_review param isn't set